dashboard/app/models/pd/application/application_base.rb

```diff
@@ -51,8 +53,10 @@ def set_type_and_year
       waitlisted
     ).index_by(&:to_sym).freeze
 
-    has_one :regional_partner
+    belongs_to :user
+    belongs_to :regional_partner
```

Comment:
This should have been [belongs_to](http://guides.rubyonrails.org/association_basics.html#the-belongs-to-association) all along. `has_one` is the inverse.